### PR TITLE
docs: update the quickstart

### LINKED ISSUE #323: Docs: update quickstart docs to work without webpack
### Background
To make the quickstart even quicker, and to make our time to awesome even lower, update the docs to work without requiring webpack, or writing any code. All the quickstart code should work by copying / pasting and opening an index.html file.

### Acceptance Criteria
The quickstart doesn't require building a webpack bundle to operate. It can be done in a single `index.html` file

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Giraffe quickstart</title>
  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
  <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
  <script crossorigin src="https://cdn.example.com/giraffe.js"></script>
</head>

<body id="home">
  <main id="root"></main>
  <script type="text/javascript">
    ReactDOM.render(
      React.createElement(Plot, {config}, null),
      document.getElementById('root')
    );
  </script>
</body>
</html>
```

### PR DESCRIPTION
**Pull request by hoorayimhelping**
Closes #323

Fills out the quickstart and makes it a little more of an easier experience.

Tested with https://dillinger.io/. Just paste the raw markdown readme file to see a rendered preview.

### CODE REVIEW

**Comment by TCL735** [comment]
Do we need a version of this for those who install giraffe via package.json and import it in their javascript or typescript files?

**Comment by hoorayimhelping** [comment]
>Do we need a version of this for those who install giraffe via package.json and import it in their javascript or typescript files?

There is a ticket to add a deeper getting started guide that would cover integrating this into your application.

**Review by TCL735 — APPROVED** [review]

File: giraffe/README.md [hunk 1]
```diff
@@ -11,72 +11,155 @@ A React-based visualization library powering the data visualizations in [InfluxD
...
+    </head>
+
+    <body id="home">
+      <main id="root"></main>
+    </body>
+    </html>
+    ```
+    Save this file as `index.html`.
+
+1. Add script imports of React, React-DOM, and Giraffe to the bottom of the `head` section.
```
Comment (Palakp41) [inline giraffe/README.md:32]: I think these these bullet points should be 1., 2., 3.

Comment (hoorayimhelping) [reply giraffe/README.md:32]: @Palakp41 markdown renders consequent `1.` as a numbered list, try it out:

1. this is `1.`
1. this is `1.`
1. this is `1.`
1. this is `1.`

**Review by Palakp41 — APPROVED** [review]
LGTM

File: giraffe/README.md (hunk 1 above)
Comment (Palakp41) [reply giraffe/README.md:32]: Oh cool, I didn't know that

**Pull request merged**